### DISCUSSION TITLE
Export values instead of JSON for M2M relationships

### DISCUSSION THREAD

**Discussion by nodeworks** [directus/directus#12288] [Feature Requests]
if you export an M2M relationship in CSV format it displays the JSON in the cell instead of the actual value. Is it possible to get just the value?

For example it displays as:

[{"regions_id":{"short_name":"SOMEREGION"}}, {"regions_id":{"short_name":"SOMEOTHERREGION"}}]

Would it be feasible to have it be:

"SOMEREGION,SOMEOTHERREGION"

**Comment by rijkvanzanten** [comment]
We should think through how this would work when the nested value isn't a single value path, but instead an object with multiple properties 🤔 Technically speaking, you could export on `regions_id` rather than `regions_id.short_name` in which case you'd still get the json object representation

**Reply by nodeworks** [reply]
I think we'd have to add a custom transform before the CSVTransforms.flatten({ separator: '.' }) transform perhaps? that way we can have flexibility in passing options to flatten.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
